src/context/cart-context/cart-context.js

```diff
@@ -0,0 +1,93 @@
+import { useContext, useState, createContext } from "react";
+import { useEffect } from "react";
```

Comment:
You missed to call useEffect in line 1 itself.

Reply:
will check